### ISSUE TITLE
multi-label behaves differently from single-label

### ISSUE THREAD

**Issue by alext234** [facebookresearch/fastText#898]
* with this training data there is multiple label

```
_label__greet Good Morning
__label__greet Good Evening
__label__greet Good Day
__label__greet Good Afternoon
__label__greet All the best
__label__greet Good luck
__label__greet Happy Birthday
__label__greet Happy Journey
__label__wish __label__question Good Morning. Did you have break-fast ?
__label__question When did you come ?
__label__question When did you reach office ?
__label__question Where did you go in the morning ?
__label__question What did you bring for lunch ?
```
train the model
```
fasttext supervised -input train.txt -output model -epoch 1000
```
and predict 

```
> fasttext predict-prob model.bin - 2
Good Morning. Did you have break-fast ?
__label__question 0.549177 __label__wish 0.441787
```

Note that **__label__question 0.549177**

* Now we the same training data, I make it single-label by removing the ___label__wish_
```
_label__greet Good Morning
__label__greet Good Evening
__label__greet Good Day
__label__greet Good Afternoon
__label__greet All the best
__label__greet Good luck
__label__greet Happy Birthday
__label__greet Happy Journey
__label__question Good Morning. Did you have break-fast ?
__label__question When did you come ?
__label__question When did you reach office ?
__label__question Where did you go in the morning ?
__label__question What did you bring for lunch ?
```
train the model
```
fasttext supervised -input train.txt -output model -epoch 1000
```

And predict
```
fasttext predict-prob model.bin -
Good Morning. Did you have break-fast ?
__label__question 0.997729
```

Note that now **__label__question 0.997729**

Why is the predicted prob for  __label__question different in these cases?

**Comment by Celebio** [comment]
Hi @alext234 ,
You are using the default loss, which is `softmax`. It means that the probabilities of a prediction sum to 1.

If you want the prediction of each label to be independent from the others, you can use the loss one-vs-all with `-loss one-vs-all` (or `-loss ova`) as described [here](https://fasttext.cc/docs/en/supervised-tutorial.html#multi-label-classification).

Best regards,
Onur

**Comment by alext234** [comment]
Cool! that helps !
